packages/mui-material-next/src/MenuItem/MenuItem.js

```diff
@@ -4,17 +4,19 @@ import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { unstable_composeClasses as composeClasses } from '@mui/base/composeClasses';
 import { alpha } from '@mui/system';
+import { useSlotProps } from '@mui/base';
+import { rootShouldForwardProp } from '@mui/material/styles/styled';
+import { styled, useThemeProps } from '@mui/material/styles';
+import { useMenuItem } from '@mui/base/useMenuItem';
+import ListContext from '@mui/material/List/ListContext';
+import ButtonBase from '@mui/material/ButtonBase';
 import {
   unstable_useEnhancedEffect as useEnhancedEffect,
   unstable_useForkRef as useForkRef,
 } from '@mui/utils';
-import ListContext from '@mui/material/List/ListContext';
-import ButtonBase from '@mui/material/ButtonBase';
+import { dividerClasses } from '@mui/material/Divider';
```

Comment:
The Divider was recently added to `material-next`: https://github.com/mui/material-ui/pull/39179